spec/models/app_setting_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'rails_helper'
 
+# We should add some tests here, which test that the validation is correct
```

Comment:
This should be the last thing for this PR, right?

Reply:
Correct!

Reply:
Done :)